llpc/context/llpcCompiler.h

```diff
@@ -102,6 +102,13 @@ class Compiler : public ICompiler {
 
   virtual Result BuildShaderModule(const ShaderModuleBuildInfo *shaderInfo, ShaderModuleBuildOut *shaderOut);
 
+  virtual Result buildGraphicsShaderStage(const GraphicsPipelineBuildInfo *pipelineInfo,
+                                          GraphicsPipelineBuildOut *pipelineOut, Vkgc::UnlinkedShaderStage stage,
+                                          void *pipelineDumpFile = nullptr);
+
+  virtual Result buildGraphicsPipelineWithElf(const GraphicsPipelineBuildInfo *pipelineInfo,
+                                              GraphicsPipelineBuildOut *pipelineOut, const BinaryData *elfPackage);
+
```

Comment:
Do you have a testing strategy for these two functions? `buildUnlinkedShaderInternal` should be testeded using the relocatable shader tests.  However the new public functions should be tested, I'm just not sure how extensively.

Reply:
For `buildGraphicsShaderStage`, we will generate a part pipeline file which only includes  relevant shaders information. For  `buildGraphicsPipelineWithElf`, we don't have a good solution yet, currently only test it with cts.